### ISSUE TITLE
multiselect check boxes don't toggle in yeoman generated angular project using bower. 

### ISSUE THREAD

**Issue by ryan4888** [davidstutz/bootstrap-multiselect#633]
I added the dependency to my bower.json, then ran bower update. I am attempting to display one of the examples. Here is the code. 

    <script type="text/javascript">
        $(function () {
            $('#lstFruits').multiselect({
                includeSelectAllOption: true
            });

        });
    </script>
    <select id="lstFruits" multiple="multiple">
       <option value="cheese" selected>Cheese</option>
        <option value="tomatoes">Tomatoes</option>
        <option value="mozarella">Mozzarella</option>
        <option value="mushrooms" selected>Mushrooms</option>
        <option value="pepperoni" selected>Pepperoni</option>
        <option value="onions">Onions</option>
    </select>

If the values in the list are predefined to be checked, then they appear checked. However I can not uncheck them. If values are not predefined to be checked, then they simply don't work. 

In short, I am unable to toggle the check boxes in the list.

**Comment by davidstutz** [comment]
From that description I can't explain the error, can you provide somr more information. You can re-open if the issue persists.
